crypto/hmac/hmac_lcl.h

```diff
@@ -10,13 +10,6 @@
 #ifndef HEADER_HMAC_LCL_H
 # define HEADER_HMAC_LCL_H
 
-#ifdef  __cplusplus
-extern "C" {
-#endif
-#if 0                            /* emacs indentation fix */
-}
-#endif
-
```

Comment:
What was this? I went ahead and removed it, should I check if there are other similar emacs specific tricks in the code??

Reply:
I had forgotten about those. That's so emacs and other indentation programs don't add extra spaces on everything that's between

``` C
extern "C" {
```

and the corresponding

``` C
} 
```

We should actually use this trick anywhere we have these C++ lines, that would save some cuss words...

Reply:
It's fine to remove that trick when you remove the C++ stuff, of course

Reply:
I think it's the only location.

Reply:
My Vim is different! 😝

Reply:
No vim-vs-emacs flames, please ;-)

Reply:
> No vim-vs-emacs flames, please ;-)

Oh no, I was already making popcorn!!

Reply:
#6557 means we won't need that kind of trick any more.